File: src/codingChallenges/PalindromeString.java
package codingChallenges;

public class PalindromeString {

    public static void main(String[] args) {
        String s = "level";

        System.out.println(palindrome(s));
    }

    private static boolean palindrome(String s) {

        Boolean palindrome = false;
        StringBuilder reverse = new StringBuilder();

        for (int i = s.length() - 1; i >= 0; i--) {
            reverse.append(s.charAt(i));
        }

        if (s.equals(reverse.toString())) {
            palindrome = true;
        }

        return palindrome;
    }


}

File: src/codingChallenges/Python/PalindromeString.py
s = "kayak"


class Solution(object):
    def __init__(self, s):
        self.s = s

    def palindrome(self, s):
        reverse = s[::-1]

        if (reverse == s):
            return True
        else:
            return False


res = Solution(s)
print(res.palindrome(s))
